Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/README.md

```diff
@@ -44,9 +43,10 @@ The use-case is designed for flux-tunable transmon qubits where the qubit-qubit
 
 ```python
 def bake_cz(baker: Baking, q1, q2):
-    q1_xy_element = f"q{q1}_xy" #
-    q2_xy_element = f"q{q2}_xy"
-    q1_z_element = f"q{q1}_z"
+    q1_xy_element = f"q{q1_idx_str}_xy"
+    q2_xy_element = f"q{q2_idx_str}_xy"
+    q1_z_element = f"q{q1_idx_str}_z"
```

Comment:
Same here, in that it is taken globally. These variables are simply defined above somewhere in the example script.

Reply:
Yeah I see the idea, I am just a bit concerned because people may want to define these function in a separated file because they will be fixed, and change only the qubit indices in the main.
I think it is fine for the use-case, but should be changed when putting the 2 qbt RB in the main folder